### DISCUSSION TITLE
7. February 2023

### DISCUSSION THREAD

**Discussion by dimakuv** [gramineproject/gramine#1157] [Meeting notes]
### Agenda

*(please write your proposed agenda items in comments under this discussion)*

- [ ] Dmitrii: Forking in multithreaded apps, need a fix (see https://github.com/gramineproject/gramine/issues/1156)
  -  **skipped**, may be discussed in next meeting
- [ ] Dmitrii: Customer request for merging https://github.com/gramineproject/gramine/pull/983
  -  **skipped**, as it was discussed in the PR itself
- [x] Mona: Review Gramine CCC update slides
- [x] Anees: Packaging and `apt update gramine-dcap` problem

#### Items not in agenda

- Release: Woju needs to get to his machine with the signing key. Will probably happen on Thursday.

#### Mona: Review Gramine CCC update slides

[ Mona showed the slides to be presented at the CCC meeting ]

- CCC is not interested in the status of the Gramine project itself.
- CCC is only interested in the Technical Charter, operational things, etc.
- So Mona wants to show the following: intro, customers, deployed use cases, progress since last update, talks & blogs, operational things (technical charter, CoC), mode of operation (Zoom, Gitter, Google groups, Jenkins CI), releases.

#### Anees: Packaging and `apt update gramine-dcap` problem

- Anees: packaging for the release. What do we do in case of `apt update gramine-dcap`?
  - Woju: I need to fix it.
  - Woju: there are new instructions for people who are upgrading in the Release Notes for Gramine v1.4.
  
- Dmitrii: how to install older Gramine packages?
  - Woju: it is possible to do `https://packages.gramineproject.io/ 1.3 main` (i.e., exact version instead of `bionic`/`bullseye`/etc). But there are caveats with OS distro versions and dependencies' versions! Like, v1.3.1 can only be easily installed on Ubuntu 22.04.
  - Woju: another workaround is to download .deb files and install them manually, resolving dependencies manually.
  - Woju: the final solution is to build Gramine manually, via `git clone ... meson setup ... ninja build install`. This is the best solution actually.

**Comment by monavij** [comment]
Review Gramine CCC update slides - we are scheduled to provide and update to CCC TAC on 2/9
